http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreterSpec.scala

```diff
@@ -221,6 +225,62 @@ class AKSInterpreterSpec extends AnyFlatSpecLike with TestComponent with Leonard
       res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
     }
 
+  it should s"exit with app in Updating status if error occurs in helm client" in isolatedDbTest {
```

Comment:
I would have expected the app to transition to Error in this case. The only time I see an app getting stuck in `updating` would be when leo restarts during the update, but handling this is outside the scope of the PR

Reply:
It does transition to error, but the handling (and test) for that is in the `LeoPubsubMessageSubcriber`, as it applies to both google and azure

Reply:
Ohhh gotcha, I missed that initially. Thanks! Adding a little comment in the test to that effect could be useful for future readers